bylaws.html

```diff
@@ -252,80 +250,74 @@ <h3 id="voting">3.3. Voting</h3>
-For major changes to the fuctionality of CouchDB, a Request For Comment (RFC) process has been established. The intent of an RFC is to ensure sufficient public discussion has occurred on the design of new features, that the proposal is adequately captured in a summary document, and that there is community acceptance of the proposal. The completed RFC template for the proposal then becomes the basis for the new functionality's documentation.
+<p>We have established a Request For Comment (RFC) process for major changes to the functionality of CouchDB. An RFC intends to ensure sufficient discussion has occurred, that the proposal is adequately captured in a summary document, and that there is community acceptance of the plan. The completed RFC template then becomes the basis for the new functionality's documentation.</p>
 
-The process is:
+<p>The process is:</p>
 
   <ul>
     <li>Start a [DISCUSS] thread on <a href="https://lists.apache.org/list.html?dev@couchdb.apache.org">the developer mailing list</a>. Discuss your proposal in detail, including which modules/applications are affected, any HTTP API additions and deprecations, and security issues.</li>
-    <li>When there is consensus on the approach from the community, <a href="https://s.apache.org/CouchDB-RFC">complete the RFC template</a> and work through any final revisions to the document, with the support of the developer mailing list.</li>
-    <li>Start the RFC <strong>vote</strong> on the developer mailing list. Hold the vote according to the <em>lazy majority process</em>: at least 3 +1 votes, and more binding +1 than binding -1 votes.</li>
+    <li>When there is consensus on the approach, <a href="https://s.apache.org/CouchDB-RFC">complete the RFC template</a>, and work through any final revisions to the document with the support of the developer mailing list.</li>
+    <li>Start the RFC <strong>vote</strong> on the developer mailing list. Hold the vote according to the <em>lazy majority process</em>: at least 3 +1 votes, and more binding +1 votes than binding -1 votes.</li>
   </ul>
 
 <h3 id="api">3.7 API changes and deprecations</h3>
 
-<p>Whenever a change is proposed to a release branch or <code>master</code> that removes or changes an existing HTTP API endpoint in a way that breaks backwards compatibility with a released version of CouchDB, the proposed change <strong><em>must</em> be announced on <a href="https://lists.apache.org/list.html?dev@couchdb.apache.org">the developer mailing list</a>, and a minimum of a <a href="#lazy">lazy consensus</a> decision is required.</strong>
+<p>Before you make a change to a release branch that removes or changes an existing HTTP API endpoint in a way that breaks backward compatibility with a released version of CouchDB,  you <em>must</em> propose the change on the <a href="https://lists.apache.org/list.html?dev@couchdb.apache.org">developer mailing list</a>. A minimum of a <a href="#lazy">lazy consensus</a> is required to proceed.</strong>
```

Comment:
-1. This changes the semantics of the original because you've left the `master` branch out. I disagree with that policy change, and if it was intentional, we need to discuss it on the list.